servant-server/servant-server.cabal

```diff
@@ -73,7 +73,7 @@ library
   -- strict dependency as we re-export 'servant' things.
   build-depends:
       servant             >= 0.19     && < 0.20
-    , http-api-data       >= 0.4.1    && < 0.4.4
+    , http-api-data       >= 0.4.1    && < 0.5.1
```

Comment:
```suggestion
    , http-api-data       >= 0.4.1    && < 0.6
```

Reply:
Updated.

Reply:
It appears you only updated the bounds for `servant` but not `servant-server`.